generator.go

```diff
@@ -115,6 +116,13 @@ func WithAddStructTagName(structTagName string) GeneratorOption {
 	}
 }
 
+// WithAddImport add custom package import.
+func WithAddImport(cImport string) GeneratorOption {
```

Comment:
I would change the API here to be `WithImports(imports ...string)`. This allows the user to add either a single or multiple imports with a single option.